src/components/Footer.tsx

```diff
@@ -0,0 +1,62 @@
+const currentYear = new Date().getFullYear();
+
+export function Footer() {
+    return (
+        <footer className="relative w-full bg-navbar py-3">
+            <div className="mx-auto w-full px-5 text-center">
+                <div className="gap-4">
+                    <div className="text-white font-bold">WEB3 WINES</div>
+                    <div>
+                        <p>Address: 1 Eagle St., Brisbane QLD 4000</p>
+                        <p>Email: info@web3wines.org</p>
+                        <p>Phone: (04)2345678</p>
```

Comment:
Probably remove this line or add a real phone number.

Reply:
issue fixed